.dagger/build/builder.go

```diff
@@ -210,6 +210,10 @@ func (build *Builder) Engine(ctx context.Context) (*dagger.Container, error) {
 		return nil, fmt.Errorf("unsupported engine base %q", build.base)
 	}
 
+	if build.version != "" {
+		base = base.WithAnnotation(distconsts.OCIVersionAnnotation, build.version)
```

Comment:
This is what's causing the CI failure:

```
build/builder.go:214:15: base.WithAnnotation undefined (type *dagger.Container has no field or method WithAnnotation)
```

It's expected since we just merged `WithAnnotation` and CI is still running the latest released version